### DISCUSSION TITLE
responsiveness on home page is breaking

### DISCUSSION THREAD

**Discussion by iagocarmona** [community/community#55736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Responsiveness on home page is breaking when I zoom in the page

This image shows where are breaking
- in  this case I'm with 150% of zoom in 
![image](https://github.com/community/community/assets/69121686/244848a9-efa0-4654-a6df-27dbb676484d)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @iagocarmona! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. In the meantime, can you tell us a little more about the issue that you're seeing? For example, do you only see this when zooming in on the web page using your computer or does this issue occur when viewing the web page on mobile? Does this only occur when using a particular browser? Information like that can help members in the community suggest a possible solution for you. 

Good luck!

**Comment by iagocarmona** [comment]
It was resolved
